app/connectors/ApplicationClient.scala

```diff
@@ -183,6 +183,15 @@ trait ApplicationClient {
       case _: NotFoundException => throw new OnlineTestNotFound()
     }
   }
+
+  def startPhase3Test(launchpadInviteId: String)(implicit hc: HeaderCarrier): Future[Unit] = {
+    http.PUT(s"${url.host}${url.base}/launchpad/$launchpadInviteId/markAsStarted", "").map(_ => ())
+  }
+
+  def completePhase3TestByToken(token: String)(implicit hc: HeaderCarrier): Future[Unit] = {
+    http.PUT(s"${url.host}${url.base}/launchpad/$token/markAsComplete", "").map(_ => ())
```

Comment:
If we are going to put the token on the URL to the backend, can we ensure that we validate it doesn't contain special characters, or escape them?